crates/matrix-sdk-crypto/src/gossiping/machine.rs

```diff
@@ -826,74 +829,71 @@ impl GossipMachine {
     async fn receive_secret(
         &self,
         sender_key: Curve25519PublicKey,
-        event: &DecryptedSecretSendEvent,
-        request: &GossipRequest,
-        secret_name: &SecretName,
+        secret: GossippedSecret,
+        changes: &mut Changes,
     ) -> Result<(), CryptoStoreError> {
-        debug!(
-            request_id = event.content.request_id.as_str(),
-            secret_name = secret_name.as_ref(),
-            "Received a m.secret.send event with a matching request"
-        );
+        debug!("Received a m.secret.send event with a matching request");
 
         if let Some(device) =
-            self.inner.store.get_device_from_curve_key(&event.sender, sender_key).await?
+            self.inner.store.get_device_from_curve_key(&secret.event.sender, sender_key).await?
         {
             // Only accept secrets from one of our own trusted devices.
             if device.user_id() == self.user_id() && device.is_verified() {
-                self.accept_secret(event, request, secret_name).await?;
+                self.accept_secret(secret, changes).await?;
             } else {
-                warn!(
-                    request_id = event.content.request_id.as_str(),
-                    secret_name = secret_name.as_ref(),
-                    "Received a m.secret.send event from another user or from \
-                    unverified device"
-                );
+                warn!("Received a m.secret.send event from another user or from unverified device");
             }
         } else {
-            warn!(
-                request_id = event.content.request_id.as_str(),
-                secret_name = secret_name.as_ref(),
-                "Received a m.secret.send event from an unknown device"
-            );
-            self.inner.store.mark_user_as_changed(&event.sender).await?;
+            warn!("Received a m.secret.send event from an unknown device");
+
+            self.inner.store.mark_user_as_changed(&secret.event.sender).await?;
         }
 
         Ok(())
     }
 
+    #[instrument(skip_all, fields(sender_key, sender = %event.sender, request_id = %event.content.request_id, secret_name))]
```

Comment:
```suggestion
    #[instrument(skip_all, fields(sender_key, sender = ?event.sender, request_id = ?event.content.request_id, secret_name))]
```

Not sure about the request ID, but for Ruma IDs we have standardized on the debug formatting.

Reply:
Gah, right, fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/2227/commits/dcd08ce2f411a5d2e5bf8c4f996c58eb5260d56d.